.github/pull_request_template.md

```diff
@@ -10,6 +10,11 @@ By submitting this pull request, I confirm that you can use, modify, copy, and r
 # Tests
 _Describe what tests you have done._
 
+#Requirements
+_Before commit the code, please do the following steps._
+1. Run `make fmt` and `make fmt-sh`
+2. Run [golangci-lint run](https://golangci-lint.run/usage/install/)
```

Comment:
I don't think that we should force people to need to run this external lint. Why would we need to do this if we are already requiring `make fmt`? The checks should be consistent.

Reply:
`make fmt` only formats code, while `golangci-lint` would do other thing such as security check, simplifying it as well. One thing we could do to make consistency is remove `goimport` and `gofmt` as default linter in `golangci-lint`

Reply:
Can we discuss the necessity and benefits of this? I would much more prefer only need to run build/make which have gofmt embedded and not worry about those two extra steps added here. I think there're lint and security check we can add in Github for PRs, so you won't need to do it locally.

Reply:
Some are not having with `github-action` such as `simply`, `govet`, `gosec`, `misspell` based on my finding so far. However, we can inject the `golangci-lint` to `Makefile` as a binary (same as GoImport if that make sense)

Reply:
Adding installing binaries in the Makefile for `golangci-lint`, so we could use `make linter` to use the golangci-linter without any specification.